app/dashboard/router.py

```diff
@@ -259,7 +259,10 @@ def get_queryset(self):
         queryset = queryset.distinct()
 
         # offset / limit
-        limit = self.request.query_params.get('limit', None)
+        limit = self.request.query_params.get('limit', 20)
```

Comment:
This is what I was thinking as well...  But in this case, how are we handling paging through the results, for example in the explorer?

Reply:
explicitly set the limit to 500 in the explorer until the pagination PR is merged

Reply:
The explorer is already equipped to step through each limit up to 500 incrementing the requesting offset?

Reply:
no it doesnt make new requests as you scroll down the page.  it loads them all initially into a client side cache and them paints them as you scroll down the page
b2e29bc0bbfd5d86fd708b59b235480f3bda51c8